tests/src/test/java/com/cloudera/director/google/compute/GoogleComputeProviderFullCycleTest.java

```diff
@@ -106,6 +107,15 @@ public GoogleComputeProviderFullCycleTest(String localSSDInterfaceType, String i
 
   @Test
   public void testFullCycle() throws InterruptedException, IOException {
+    doTestFullCycle(null);
+  }
+
+  @Test
+  public void testFullCyclePreemptible() throws InterruptedException, IOException {
+    doTestFullCycle(true);
+  }
+
+  public void doTestFullCycle(Boolean usePreemptibleInstances) throws InterruptedException, IOException {
```

Comment:
What's the reason behind using the wrapper `Boolean` instance of `boolean` and dropping the null-check?